Makefile

```diff
@@ -46,4 +49,17 @@ ineffassign:
 run: build
 	$(shell bin/vegeta-server --scheme=http --host=localhost --port=8000)
 
+
+container:
+	docker build -t vegeta-server:latest .
+
+container_run:
```

Comment:
change this to `container_run: container`, which should ensure that the `docker build` is called as a precursor to `docker run`.